tests/eas/offload_idle_pull.config

```diff
@@ -0,0 +1,50 @@
+{
+    "modules"  : [ "bl" ],
+    "tools"    : [ "rt-app" ],
+    "ftrace" : { // TODO make these inherited
```

Comment:
How do you want to inherit these?
In any case, if we do not do it now let's remove the comment as well from the committed code.